test/ValidateID.cpp

```diff
@@ -406,7 +406,7 @@ class OpTypeArrayLengthTest
     : public spvtest::TextToBinaryTestBase<::testing::TestWithParam<int>> {
  protected:
   OpTypeArrayLengthTest()
-      : position_{0, 0, 0}, diagnostic_(spvDiagnosticCreate(&position_, "")) {}
+      : position_(), diagnostic_(spvDiagnosticCreate(&position_, "")) {}
```

Comment:
I think we should leave it in the initializer list.  Otherwise the compiler might not initialize it.  In some flows that's considered a bug, and I'd agree with it.
I believe (should look it up), using () to initialize it will mean it's zero-initialized, which is the right thing.